src/main/scala/com/spotify/spark/bigquery/BigQueryClient.scala

```diff
@@ -109,8 +110,29 @@ private[bigquery] class BigQueryClient(conf: Configuration) {
    */
   def load(gcsPath: String, destinationTable: TableReference,
            writeDisposition: WriteDisposition.Value = null,
-           createDisposition: CreateDisposition.Value = null): Unit = {
+           createDisposition: CreateDisposition.Value = null,
+           isPartitionedByDay: Boolean = false): Unit = {
```

Comment:
Guys what would you like to do? Merge this and work on exposing the client or Close it altogether?